public/src/components/epicTests/epicTestVariantsList.tsx

```diff
@@ -111,23 +116,34 @@ class EpicTestVariantsList extends React.Component<EpicTestVariantsListProps, Ep
                 <EpicTestVariantEditor
                   variant={variant}
                   onVariantChange={this.onVariantChange}
+                  editMode={this.props.editMode}
                 />
               </ExpansionPanelDetails>
             </ExpansionPanel>
           );
         })}
       </>
     );
-  }
+  };
 
-  render(): React.ReactNode {
+  renderVariants = (): React.ReactNode => (
```

Comment:
this doesn't need to be a separate function

Reply:
Ah ok will take a look

Reply:
I've updated the file now.